sim/rogue/runes.go

```diff
@@ -153,3 +157,132 @@ func (rogue *Rogue) ApplyCutToTheChase(sim *core.Simulation) {
 		rogue.SliceAndDiceAura.Activate(sim)
 	}
 }
+
+// TODO: Update cut to the chase to refresh
+func (rogue *Rogue) registerBladeDance() {
+	if !rogue.HasRune(proto.RogueRune_RuneBladeDance) {
+		return
+	}
+
+	justAFleshWound := rogue.HasRune(proto.RogueRune_RuneJustAFleshWound)
+
+	rogue.bladeDanceDurations = [6]time.Duration{
+		0,
+		time.Duration(time.Second * 14),
+		time.Duration(time.Second * 18),
+		time.Duration(time.Second * 22),
+		time.Duration(time.Second * 26),
+		time.Duration(time.Second * 30),
+	}
+
+	rogue.BladeDanceAura = rogue.RegisterAura(core.Aura{
+		Label:    "Blade Dance",
+		ActionID: core.ActionID{SpellID: int32(proto.RogueRune_RuneBladeDance)},
+		Duration: rogue.bladeDanceDurations[5],
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+			rogue.AddStatDynamic(sim, stats.Parry, 10*core.ParryRatingPerParryChance)
+			if justAFleshWound {
+				rogue.PseudoStats.SchoolDamageTakenMultiplier[stats.SchoolIndexPhysical] *= 0.8
+			}
+		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+			rogue.AddStatDynamic(sim, stats.Parry, -10*core.ParryRatingPerParryChance)
+			if justAFleshWound {
+				rogue.PseudoStats.SchoolDamageTakenMultiplier[stats.SchoolIndexPhysical] /= 0.8
+			}
+		},
+	})
+
+	rogue.BladeDance = rogue.RegisterSpell(core.SpellConfig{
+		ActionID:     core.ActionID{SpellID: int32(proto.RogueRune_RuneBladeDance)},
+		SpellSchool:  core.SpellSchoolPhysical,
+		Flags:        core.SpellFlagAPL, // TODO: Add all flags
+		MetricSplits: 6,
+
+		EnergyCost: core.EnergyCostOptions{
+			Cost: 25,
+		},
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD: time.Second,
+			},
+			IgnoreHaste: true,
+			ModifyCast: func(s1 *core.Simulation, spell *core.Spell, cast *core.Cast) {
+				spell.SetMetricsSplit(spell.Unit.ComboPoints())
+			},
+		},
+		ExtraCastCondition: func(sim *core.Simulation, target *core.Unit) bool {
+			return rogue.ComboPoints() > 0
+		},
+
+		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+			rogue.BladeDanceAura.Duration = rogue.bladeDanceDurations[rogue.ComboPoints()]
+			rogue.BladeDanceAura.Activate(sim)
+			rogue.ApplyFinisher(sim, spell)
+		},
+	})
+}
+
+func (rogue *Rogue) applyJustAFleshWound() {
+	if !rogue.HasRune(proto.RogueRune_RuneJustAFleshWound) {
+		return
+	}
+	// Mod threat
+	// TODO: Confirm threat mod
+	rogue.PseudoStats.ThreatMultiplier *= 1.895
+
+	// Blade Dance 20% Physical DR - Added in registerBladeDance()
+
+	// -6% to be critically hit
+	rogue.PseudoStats.ReducedCritTakenChance += 6
+
+	// Replace Feint with Tease
+	// Warrior sim from wrath did not implement it. May implement later
```

Comment:
Could you add a TODO on this comment if you're not planning to include it for now?